django/core/serializers/xml_serializer.py

```diff
@@ -208,16 +208,18 @@ def _make_parser(self):
     def __next__(self):
         for event, node in self.event_stream:
             if event == "START_ELEMENT" and node.nodeName == "object":
+                # Look up the model using the model loading mechanism. If this fails,
+                # bail.
+                Model = self._get_model_from_node(node, "model")
+                if not router.allow_migrate_model(self.db, Model):
+                    continue
```

Comment:
Why it's not in `_handle_object()`?

Reply:
We lack access to Model within _handle_object(). Even if we attempt to access it as done above, we can't employ `continue` since it's not within the loop. Returning `None` would create inconsistency with other implementations (e.g., Python, JSON, YAML, etc.). This, basically, aligns with other implementations without major changes.

Reply:
> We lack access to Model within _handle_object()

`Model = self._get_model_from_node(node, "model")` is taken directly from `_handle_object()` so I'm not sure what do you mean by that.

> Returning None would create inconsistency with other implementations (e.g., Python, JSON, YAML, etc.).

You can handle this in `__next__`:
```python
if (object := self._handle_object(node, Model)) is None:
    continue
return object
```
I don't see any inconsistency here.

Reply:
Ah,  Right, we can do this. I overlooked `return self._handle_object(node)`.
Thanks